C/Program-Timer/README.md

```diff
@@ -0,0 +1,12 @@
+# Program Timing
+This script checks the time required to run a function in C.
+
+
+## To use
+* Edit `void function` and write your own operations.
+* Compile with `gcc timer.c -o timer`.
+* Execute with `./timer`
+
+
```

Comment:
please add space and time complexity.

Reply:
Hello @yashaswibiyahut, I am not sure what you mean by this. Do you want me to find the time and space complexity of `function` programatically?

Reply:
add simple space and time complexity of your **program**